webapp/src/modules/mortgage/utils.js

```diff
@@ -31,13 +31,17 @@ export function getLoanMetadata() {
 }
 
 export function isMortgageActive(mortgage, parcel) {
-  return (
-    mortgage &&
-    parcel &&
-    ((isMortgagePending(mortgage) &&
-      isOpen(parcel.publication, PUBLICATION_STATUS.open)) ||
-      hasStatus(mortgage, [MORTGAGE_STATUS.ongoing, MORTGAGE_STATUS.paid]))
-  )
+  if (mortgage && parcel) {
```

Comment:
What if we first make validations and then the business logic? So we don't mix the operations with validations, in this case we can do something like
```js
if (!mortgage || !parcel) {
	return
}

// do stuff